assets/js/state/sagas/softwareUpdates.js

```diff
@@ -21,10 +23,20 @@ export function* fetchSoftwareUpdates({ payload: hostID }) {
   try {
     const response = yield call(getSoftwareUpdates, hostID);
     yield put(setSoftwareUpdates({ hostID, ...response.data }));
+    yield put(setSettingsConfigured());
   } catch (error) {
     yield put(setEmptySoftwareUpdates({ hostID }));
 
-    const errors = get(error, ['response', 'data'], []);
+    const errorCode = get(error, ['response', 'status']);
+    const { errors } = get(error, ['response', 'data'], []);
+    const suma_unauthorized = errors.some(
+      ({ detail }) => detail === 'SUSE Manager authentication error.'
+    );
+
+    if (errorCode === 422 && suma_unauthorized) {
```

Comment:
I check the status code here for completeness, though technically we may not need to.